scripts/upgrade

```diff
@@ -87,9 +87,14 @@ fi
 
 # Flows were stored in file named after the hostname.
 # Not very portable. Let's fix that.
-if [ ! -f "$final_path/data/flows.json" ]; then
+if [[ ! -f "$final_path/data/flows.json" && -f "$final_path/data/flows_$(hostname).json" ]]; then
 	mv "$final_path/data/flows_$(hostname)_cred.json" "$final_path/data/flows_cred.json"
 	mv "$final_path/data/flows_$(hostname).json" "$final_path/data/flows.json"
+# Flows could be stored in a file named '>>'.
+# Definitely weird, let's fix that.
```

Comment:
Do you have any clue why it would have been stored in such a name?

Reply:
Yup... a very badly written systemd file, ending with `>> ...` in an attempt to redirect any output to the log file.  (That's the only location I had found those two characters)